src/app/attendance/components/AttendanceFormBuilder.tsx

```diff
@@ -102,16 +105,33 @@ const AttendanceFormBuilder = () => {
     };
 
     fetchAttendance();
```

Comment:
This looks wrong, why are we fetching attendance here? Why can't this come from server component?

Reply:
Fixed